src/adjoint_common.jl

```diff
@@ -379,11 +391,11 @@ function get_paramjac_config(autojacvec::ReverseDiffVJP, p, f, y, _p, _t;
         if !isRODE
             __p = p isa SciMLBase.NullParameters ? _p :
                   SciMLStructures.replace(Tunable(), p, _p)
-            tape = ReverseDiff.GradientTape((y, __p, [_t])) do u, p, t
+            tape = ReverseDiff.GradientTape((y, _p, [_t])) do u, p, t
```

Comment:
are you sure this is the right one?

Reply:
The tunables are replaced into the original structure (`p`) from `_p` and we repack the `_p` tunables again into `p` again when passing into the `f`

Reply:
The difference is that we were passing just `tunables` into `f`, now we are passing the SciMLStructure, which is what the `f` expects. cc @AayushSabharwal for whether this makes sense

Reply:
>The difference is that we were passing just tunables into f

Makes sense.